### PR TITLE
[#401] date regex fix

### PR DESCRIPTION
**Pull request by TaimoorRana**
Added/updated regexes for dates

### CODE REVIEW

File: src/ml_service/feature_extraction/post_processing/regex/regex_lib.py [hunk 1]
```diff
@@ -7,7 +7,10 @@ class RegexLib:
     TENANT_REGEX = r"locataire(s)?"
     LANDLORD_REGEX = r"locat(eur|rice)(s)?"
     DEMAND_REGEX = r"(demand|réclam)(ait|e|ent|aient)"
-    DATE_REGEX = r"(janvier|février|mars|avril|d'avril|mai|juin|juillet|d'août|août|septembre|d'octobre|octobre|novembre|décembre)"
+    DATE_REGEX = r"(janvier|février|fevrier|mars|avril|d'avril|mai|juin|juillet|d'août|août|aout|septembre|d'octobre" \
+                 r"|octobre|novembre|décembre|decembre)"
+    DATE_RANGE_REGEX = r"(?i)(\d{1,2})?(?:er|èr|ere|em|eme|ème)?\s?(\w{3,9}) (\d{4})?\s?" \
```
Comment (Samuel-Campbell) [inline src/ml_service/feature_extraction/post_processing/regex/regex_lib.py:12]: careful. We don't want the time of the lease but the time for which a tenant hasn't paid.

**Review by Samuel-Campbell — CHANGES_REQUESTED** [review]
Regex will catch too many irrelevant things

File: src/ml_service/feature_extraction/post_processing/regex/regex_lib.py (hunk 1 above)
Comment (TaimoorRana) [reply src/ml_service/feature_extraction/post_processing/regex/regex_lib.py:12]: Oh, I didn't know we needed it specifically for that. I'll change it

Comment (Samuel-Campbell) [reply src/ml_service/feature_extraction/post_processing/regex/regex_lib.py:12]: its possible we don't even need the regex after thinking about it. Check ml channel

Comment (TaimoorRana) [reply src/ml_service/feature_extraction/post_processing/regex/regex_lib.py:12]: I saw the channel but this way almost done, so I finished it.

Comment (TaimoorRana) [reply src/ml_service/feature_extraction/post_processing/regex/regex_lib.py:12]: if we ever need it, it will be there.

**Review by Samuel-Campbell — APPROVED** [review]
can you tag precedents and upload model to CJL when you got time?

**Comment by TaimoorRana** [comment]
sure

**Pull request merged**